src/base/emu-i386/simx86/interp.c

```diff
@@ -599,8 +604,8 @@ static TNode *_Interp86(unsigned int PC, unsigned int Interp_LONG_CS,
 		if (e_querymark(PC, gap)) break;
 		do {
 			PC = InterpOne(PC, Interp_LONG_CS, ocs, basemode);
-			G = interp_post(PC, Interp_LONG_CS, basemode, flags);
-		} while (!G);
+		} while (!interp_post(PC, Interp_LONG_CS, basemode, flags));
+		G = DoClose(PC, Interp_LONG_CS, basemode, flags);
```

Comment:
But how NULL return is
handled? (which may now
happen)

Reply:
Converted `if` to `while` to try again if ever needed.